tests/core/project-test.cpp

```diff
@@ -89,6 +89,9 @@ class ProjectTestFixture : public ::testing::Test {
     MemoryManager memoryManager = projectModule.getMemoryManager();
     memoryManager.setErrorCallback(errorCallback);
     architectureValidator.validate();
+    CommandInterface commandInterface = projectModule.getCommandInterface();
+    commandInterface.setSyncCallback(
+        [this]() { this->projectModule.guiReady(); });

```

Comment:
There's special syntax that let's you write a lambda without arguments simply as `[...]{...}` (no parantheses)